src/main/java/lotto/domain/LottoMachine.java

```diff
@@ -0,0 +1,30 @@
+package lotto.domain;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.ArrayList;
+import java.util.List;
+import lotto.constant.ValidationConstant;
+
+public class LottoMachine {
+    public LottoMachine() {
+    }
+
+    public Lottos issueLottos(Money money) {
+        List<Lotto> issuedLottos = new ArrayList<>();
+        for(int i = 0; i < howManyLotto(money); i++) {
```

Comment:
이 경우 반복문의 매 실행이 끝날 때 마다 `howManyLotto()` 메서드가 호출됩니다.
해당 부분을 변수로 추출하면 성능면에서 더 좋을 것 같아요!
```suggestion
        int lottoAmount =  howManyLotto(money);
        for(int i = 0; i <lottoAmount; i++) {
```